qbft/spectest/tests/prepare/set_prepared_full_data.go

```diff
@@ -1,8 +1,36 @@
 package prepare
 
-import "github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
 
 // SetPreparedFullData tests setting state.LastPreparedValue to the full data not the root
 func SetPreparedFullData() tests.SpecTest {
-	panic("implement")
+	ks := testingutils.Testing4SharesSet()
+
+	pre := testingutils.BaseInstance()
+	pre.State.Round = 10
+
+	rcMsgs := []*qbft.SignedMessage{
+		testingutils.TestingRoundChangeMessageWithRound(ks.Shares[1], types.OperatorID(1), 10),
+		testingutils.TestingRoundChangeMessageWithRound(ks.Shares[2], types.OperatorID(2), 10),
+		testingutils.TestingRoundChangeMessageWithRound(ks.Shares[3], types.OperatorID(3), 10),
+	}
+	pre.State.ProposalAcceptedForCurrentRound = testingutils.TestingProposalMessageWithParams(
+		ks.Shares[1], types.OperatorID(1), 10, qbft.FirstHeight, testingutils.TestingQBFTRootData,
+		testingutils.MarshalJustifications(rcMsgs), nil)
+
+	pre.State.LastPreparedValue = testingutils.TestingQBFTFullData
+
+	msgs := []*qbft.SignedMessage{
+		testingutils.TestingPrepareMessageWithRound(ks.Shares[1], 1, 10),
+	}
+	return &tests.MsgProcessingSpecTest{
+		Name:          "set prepared full data",
+		Pre:           pre,
+		InputMessages: msgs,
+	}
 }
```

Comment:
I actually don't understand what state transition this tests

Reply:
Yeah. I was to ask you about it. I also don't get this test.
The instruction is to test "setting state.LastPreparedValue to the full data not the root". But this is how it is normally set. Maybe the instruction is for an outdated version. What do you think?

Reply:
What the comment says is to check that pre.State.LastPreparedValue gets updated correctly after a prepare quorum was reached...

But the happyflow test already tests this...

So I think delete this test and add a github comment to why